examples/c/gpio_advanced.c

```diff
@@ -17,7 +17,7 @@
 /* mraa header */
 #include "mraa/gpio.h"
 
-#define GPIO_PIN 6
+#define GPIO_PIN 34
```

Comment:
I think you don't need to modify this file.
This example is used to show how to use interrupt.